samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
If you can demonstrate that the grammar is objectively better at highlighting, sure. The maturity and the license might also be considered.

Can't you merge your improvements/fixes back into the original grammar?